### PR TITLE
Add logstash

### PR DESCRIPTION
**Pull request by kicferk1**
## Ticket and context

Ticket: https://dfedigital.atlassian.net/browse/CPDEL-550

### Is there anything weird that code reviewer should know?
Copied pretty much everything from R&P.
I intend to see if I get any logs from E&L in the existing logstash setup, if yes it is probably working.

### CODE REVIEW

**Comment by kicferk1** [comment]
I got some logs through (you can see them under [this long link](https://kibana.logit.io/s/51151c79-e17d-4407-be3a-01960f3f65ff/app/kibana#/discover?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2021-06-14T12:55:30.000Z',to:'2021-06-14T12:56:00.000Z'))&_a=(columns:!(_source),filters:!(),index:'8ac115c0-aac1-11e8-88ea-0383c11b333a',interval:auto,query:(language:kuery,query:'cf.app:%20%22ecf-engage-and-learn-review-pr-147%22'),sort:!()))). So I guess it works.

File: config/environments/deployed_development.rb [hunk 1]
```diff
@@ -89,10 +89,27 @@
...
+  # Use Lograge for cleaner logging
+  config.lograge.enabled = true
+  config.lograge.base_controller_class = ["ActionController::API", "ActionController::Base"]
+  config.lograge.formatter = Lograge::Formatters::Logstash.new
+  config.lograge.ignore_actions = ["ApplicationController#check"]
+  config.lograge.logger = ActiveSupport::Logger.new($stdout)
+
+  # Include params in logs: https://github.com/roidrage/lograge#what-it-doesnt-do
+  config.lograge.custom_options = lambda do |event|
+    exceptions = %w[controller action format id]
```
Comment (jamesrose) [inline config/environments/deployed_development.rb:108]: Are there any secure tokens or passwords that we might want to exclude here?

Comment (kicferk1) [reply config/environments/deployed_development.rb:108]: Hmm, all I can think of is `login_token`, which I guess we should hide - it isn't very long lived, but if someone failed to log in with a valid token due to some exception, and someone had the logs, and they tried to log in with the same token, they might be able to get in.

Other than that, I don't think we have anything relating to security in params.

**Review by ethanmills — APPROVED** [review]

File: config/environments/deployed_development.rb (hunk 1 above)
Comment (kicferk1) [reply config/environments/deployed_development.rb]: Actually, we add 
```
Rails.application.config.filter_parameters += %i[
  passw secret token _key crypt salt certificate otp ssn
]
```
in one of our initialisers, and https://guides.rubyonrails.org/configuring.html tells me it works "by partial matching regular expression", so anything with a `token` should be hidden already.

**Pull request merged**